arches/app/media/css/pages/_authentication.scss

```diff
@@ -0,0 +1,188 @@
+#container.cls-container.arches-login {
+    background: rgb(236, 238, 241);
+}
+
+.arches-login>.cls-header {
+    background: rgb(236, 238, 241);
+}
+
+.arches-login div .cls-brand:after {
+    background: rgb(236, 238, 241);
+}
+
+.cls-content.arches-signin {
+    padding-top: 20px;
+}
+
+.login-panel-header.arches-signin {
+    font-size: 28px;
+}
+
+.arches-signin-subtext {
+    padding: 0px 5px 10px 5px;
+    color: #888;
+}
+
+legend.arches-signin-subtext {
+    font-size: 1em;
+    font-weight: normal;
+    border: none;
+}
+
+.floating-label-group {
+    position: relative;
+    display: flex;
+    align-items: start;
+}
+
+.floating-label-input {
+    &::placeholder {
+        color: transparent;
+    }
+}
+
+.floating-label-input:focus {
+    outline: 1;
+    box-shadow: 1;
+    background: transparent;    
+}
+
+.floating-label-input:focus ~ .floating-label, 
+.floating-label-input:valid ~ .floating-label {
+    top: -2em;
+    font-size: 0.8em;   
+}
+
+.floating-label-input.change-password:focus ~ .floating-label, 
+.floating-label-input.change-password:valid ~ .floating-label {
+    top: -0.85em; 
+}
+
+.floating-label {
+    position: absolute;
+    padding: 0 0.5em 0 0.5em;
+    top: 1em;
```

Comment:
change the padding to `padding: 0.75em;` 
remove the `top` setting

Reply:
Without `top`, the labels in the login and new user pages are not placed properly. Was your suggestion specifically regarding the change password template?

Reply:
@johnatawnclementawn I'm not seeing that.  If I make the floating label css this:
```
.floating-label {
    position: absolute;
    padding: 0.75em;
    pointer-events: none;
    user-select: none;
    transition: 0.2s ease all;
    -moz-transition: 0.2s ease all;
    -webkit-transition:0.2s ease all;
}
```
things seem to render well.  Can you confirm?

Reply:
I can confirm, although the transitions seem less smooth - top: 0; fixes that. I can remove if necessary.